api/src/server.ts

```diff
@@ -23,16 +23,17 @@ import app from "./index";
 const env = config.get("environment");
 const port = config.get("port");
 
-app.listen(port, "0.0.0.0", (err) => {
-  if (err) {
+app.listen(port, "0.0.0.0", 0, () => {
```

Comment:
The new `try`/`catch` block looks good, but the new `0` argument for `backlog` should be left out. See the [Node.js `server.listen()` documentation](https://nodejs.org/api/net.html#serverlisten) for details. A value of `0` effectively limits us to no queue at all, whereas we just want to take whatever the default is from the environment.